C/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all
+
+#LLVM_BUILD_DIR=/path/to/llvm-build
+LLVM_BUILD_DIR=/home/user/repos/benchmarking/wasmception/build/llvm
```

Comment:
I believe the upstream LLVM has wasm target enabled by default. How do you want to handle the LLVM installation?

Reply:
This PR is old and outdated. I don't want this repo to include the wrc20.c code anymore. I think that this PR should be closed, but am waiting for feedback before I close.

This is a great question which I need advice for in the current wrc20.c repo https://github.com/poemm/C_ewasm_contracts . It is good that newer version of ubuntu have LLVM 8+ by default. But getting LLVM 8+ may be a challenge for some users. The linked repo gives instructions in the README and compilation commands in the Makefile, but, in my experience, I have had to restart the compilation several times before it finished. I think that this conversation should be moved to that repo.